src/main/java/ch/njol/skript/expressions/ExprSkullOwner.java

```diff
@@ -0,0 +1,84 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.Material;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.block.Block;
+import org.bukkit.block.Skull;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Skull Owner")
+@Description("The skull owner of a player skull.")
+@Examples({
+	"set {_owner} to the skull owner of event-block",
+	"set skull owner of {_block} to player"
+})
+@Since("INSERT VERSION")
+public class ExprSkullOwner extends SimplePropertyExpression<Block, OfflinePlayer> {
+
+	static {
+		register(ExprSkullOwner.class, OfflinePlayer.class, "(head|skull) owner", "blocks");
+	}
+
+	@Override
+	public @Nullable OfflinePlayer convert(Block block) {
```

Comment:
We can improve this by adding an init and check for literals if the block is not a skull it would error